app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -721,7 +718,8 @@ fun HikeCardFor(
       messageContent = suitableLabelText,
       styleProperties =
           HikeCardStyleProperties(
-              messageIcon = painterResource(suitableLabelIcon!!),
+              messageIcon =
+                  if (suitableLabelIcon != null) painterResource(suitableLabelIcon) else null,
```

Comment:
Suggestion: Wouldn't that be better to change `suitableLabelIcon` to `messageIcon` and do `painterResource(R.drawable.{case})` in both case to reduce this weird condition here ?

Reply:
Yup, that's better indeed